docker-compose.client.yml

```diff
@@ -10,3 +10,4 @@ services:
             - mender
         stdin_open: true
         tty: true
+        privileged: true
```

Comment:
Have no idea how docker-compose and yml works, but there is an `--device` option to docker which is "more secure" then `--privileged`. It probably does not matter in the demo environment.

But one can use `--device /dev/kvm`, to provide access to KVM inside the docker container. This is what I normally do in my local setups working with QEMU. 

But again probably do not need it in demo environment but wanted to share it.

Reply:
Here is [link](https://docs.docker.com/engine/reference/commandline/run/#add-host-device-to-container---device) to docs.

Reply:
I tried this but the problem is that if `/dev/kvm` isn't available (e.g. you're using VirtualBox), then the device cannot be mapped into the container, and it fails to launch instead of falling back to the emulated variant. I don't think it's possible to do any if logic in the docker-compose file to detect this. It would be more secure indeed, but I think we can live with the `--privileged` option since it's only a demo device.